docs/configuration.md

```diff
@@ -406,6 +406,15 @@ images:
 links:
   [ <link_config> ... ]
 
+# The part or component of the affected system that is broken.
+  [ component: <tmpl_string> ]
```

Comment:
can you remove the trailing spaces? I believe they are unneeded.

Reply:
Done (I presume you meant leading spaces rather than trailing spaces). :+1:

Reply:
Of course! Thanks for deciphering :)